sdk/cosmos/azure-cosmos/test/test_crud.py

```diff
@@ -2376,16 +2377,6 @@ def test_index_progress_headers(self):
         self.assertFalse(HttpHeaders.LazyIndexingProgress in created_db.client_connection.last_response_headers)
         self.assertTrue(HttpHeaders.IndexTransformationProgress in created_db.client_connection.last_response_headers)
 
-        lazy_coll = created_db.create_container(
-            id='test_index_progress_headers lazy_coll ' + str(uuid.uuid4()),
-            indexing_policy={'indexingMode': documents.IndexingMode.Lazy},
-            partition_key=PartitionKey(path="/id", kind='Hash')
-        )
-        created_container = created_db.get_container_client(container=lazy_coll)
-        created_container.read(populate_quota_info=True)
-        self.assertTrue(HttpHeaders.LazyIndexingProgress in created_db.client_connection.last_response_headers)
-        self.assertTrue(HttpHeaders.IndexTransformationProgress in created_db.client_connection.last_response_headers)
```

Comment:
I'm wondering if it would be a good idea to leave the "lazy" tests in place, and just change the assertion statements to confirm that mode has been changed from 'lazy' to 'consistent'.
Otherwise if the service changes again in future to, for example, reject a request that uses 'lazy' we might not find out about it.
What do you think? Maybe I'm being to paranoid :)

Reply:
I had originally tried that, but then these tests fail against the emulator, which still has the old behavior :(

Reply:
Oh that's a shame...
You could try a check like this?
https://github.com/Azure/azure-sdk-for-python/blob/f785b02cc31b4d2079f2bf5f00c0e8bee96cdfd8/sdk/cosmos/azure-cosmos/test/test_crud.py#L1870

Reply:
We'll end up with the same issue in reverse. The next emulator release will pick up this change and the emulator specific checks will fail. The emulator does not release on a schedule so could be anytime.

Reply:
Yeah.... fair point.... oh well